index.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please do not disable linters. Linters enforce cocnsisstnecy and promote good coding practices. 🤝